src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,76 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  bitten() {
+    if (this.health - 50 > 0) {
```

Comment:
```suggestion
    if (this.health - 50) {
```

Reply:
Also it would be nice to store the hp value in some const and use it whenever you need it.